docs/ionq/calibrations.md

```diff
@@ -0,0 +1,60 @@
+# IonQ API Calibrations
+
+Calibrations are snapshots of the performance of the IonQ platform
+at a given moment of time.  The IonQ quantum computers are continuously
+calibrated, and periodically measurements are taken of the performance
+of the machine.
+
+In this section we assume a `cirq.ionq.Service` object has been instantiated and is
+called `service` and `cirq` and `cirq.ionq` have been imported:
+```python
+import cirq
+import cirq.ionq as ionq
+service = ionq.Service()
+```
+See [IonQ API Service](service.md) for how to set the service up.
```

Comment:
`set the service up.` - > `set up the service`

Reply:
done